### PR TITLE
New Module: Yum config manager

### PR DESCRIPTION
**Pull request by maxamillion**

### CODE REVIEW

File: system/yum_config_manager.py [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+    rc, out, err = module.run_command(cmd)
+
+    if rc != 0:
+        module.fail_json(
+            msg="rc of commmand is non-zero",
+            cmd=args,
+            stdout=out,
+            stderr=err,
+            rc=rc,
+            changed=True,
```
Comment (alxgu) [inline system/yum_config_manager.py]: changed should be "False" in a failed case

**Comment by alxgu** [comment]
I get an error with following arguments: -a "state=enabled name=unix"
NameError: global name 'args' is not defined

Check line 121,122 and 112
NameError: global name 'run_result' is not defined.

**Comment by maxamillion** [comment]
Yeah, that was a horrible oversight on my part. I admittedly wrote this for someone, they said it worked great and asked if I would submit it upstream. However I'm starting to reconsider that because the `yum-config-manager` util doesn't seem to provide useful exit codes. It's basically always 0 even if you provide a reponame that doesn't exist on the machine.

**Comment by maxamillion** [comment]
For sake of personal sanity and keeping things idempotent and reporting honestly, I'm going to close this. I might re-open later if we can get some improvements into yum-config-manager upstream though.

**Pull request closed without merging**